test/infrastructure/docker/cloudinit/writefiles_test.go

```diff
@@ -39,101 +37,38 @@ func TestWriteFiles(t *testing.T) {
 					{Path: "baz", Content: "qux"},
 				},
 			},
-			expectedlines: []string{
-				fmt.Sprintf("%s mkdir -p .\n", prompt),
-				fmt.Sprintf("%s cat > foo << END\nbar\nEND\n", prompt),
-				fmt.Sprintf("%s mkdir -p .\n", prompt),
-				fmt.Sprintf("%s cat > baz << END\nqux\nEND\n", prompt),
+			expectedCmds: []Cmd{
+				{Cmd: "mkdir", Args: []string{"-p", "."}},
+				{Cmd: "/bin/sh", Args: []string{"-c", "cat > foo /dev/stdin"}, Stdin: "bar"},
+				{Cmd: "mkdir", Args: []string{"-p", "."}},
+				{Cmd: "/bin/sh", Args: []string{"-c", "cat > baz /dev/stdin"}, Stdin: "qux"},
 			},
 		},
-		{
```

Comment:
Removed tests are related to the removed functionality and are therefore no longer needed